modules/AWS/eks/nodegroup_launch_template/main.tf

```diff
@@ -0,0 +1,26 @@
+locals {
+  user_content = templatefile("${path.module}/templates/userdata.sh.tpl", {
```

Comment:
Maybe move this local variable to locals.tf file for consistency?

Reply:
Right, I never had locals there 3 commits ago. Moved now.